eZ/Bundle/EzPublishCoreBundle/Fragment/DirectFragmentRenderer.php

```diff
@@ -86,4 +97,38 @@ protected function getArguments(callable $controller, ControllerEvent $event): a
 
         return $arguments;
     }
+
+    /**
+     * @param string|\Symfony\Component\HttpKernel\Controller\ControllerReference $uri
+     *
+     * @throws \eZ\Publish\Core\MVC\Symfony\Templating\Exception\InvalidResponseException
+     */
+    public function render(
+        $uri,
+        Request $request,
+        array $options = []
```

Comment:
Is this our own method parameter? It doesn't seem to be used.

Reply:
It comes from interface we use as well. Currently unused in this implementation.